features/step_definitions/basic_steps.rb

```diff
@@ -0,0 +1,11 @@
+Then("the {string} should be visible") do |selector|
+  expect(page).to have_selector :css, "##{selector}"
+end
+
+When("I click on {string} link") do |link|
+  click_link_or_button link
```

Comment:
IMO **if** you specify that this should be a link (as you do in the title) then you should use `click_link*  command. Also, double check that you are not duplication this step. It feels weird to see this addition this late in the project.

Reply:
👍  There where no step def for that yet, so needed to add it. But will refactor it to click_link.